src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
```

Comment:
total what?
Maybe rename to something specific?

Reply:
The function is called calculateRentalCost, so total is the full rental amount. I don't think I should change the name of a variable